config.json

```diff
@@ -45,6 +45,72 @@
         "prerequisites": [],
         "status": "active"
       },
+      {
+        "slug": "annalyns-infiltration",
+        "name": "Annalyn's Infiltration",
+        "uuid": "8833b0c0-1008-4426-8225-1dc2b4852d52",
+        "concepts": [
+          "booleans"
+        ],
+        "prerequisites": [
+          "basics"
+        ],
+        "status": "active"
+      },
+      {
+        "slug": "weather-forecast",
+        "name": "Weather Forecast",
+        "uuid": "63415c33-658e-4722-bd81-0b80060aecc9",
+        "concepts": [
+          "comments"
+        ],
+        "prerequisites": [
+          "basics"
+        ],
+        "status": "wip"
+      },
+      {
+        "slug": "savings-account",
+        "name": "Savings Account",
+        "uuid": "13f6b5ab-f4bb-453a-a6a9-50132a1d4885",
+        "concepts": [
+          "constants"
+        ],
+        "prerequisites": [
+          "basics",
+          "comments",
+          "strings"
+        ],
+        "status": "active"
+      },
+      {
+        "slug": "blackjack",
+        "name": "Blackjack",
+        "uuid": "628308fc-34e9-46ae-9661-9e0ebceeef34",
+        "concepts": [
+          "conditionals-if",
+          "conditionals-switch"
+        ],
+        "prerequisites": [
+          "booleans",
+          "strings",
+          "numbers"
+        ],
+        "status": "active"
+      },
+      {
+        "slug": "deep-thought",
+        "name": "Deep Thought",
+        "uuid": "097176e2-5024-44e7-ab86-9bc0ec245347",
+        "concepts": [
+          "errors"
+        ],
+        "prerequisites": [
+          "constants",
+          "types"
+        ],
+        "status": "wip"
```

Comment:
This one is WIP because it's missing proper instructions, hints, tests, and a stub solution.